testdata/p4_16_samples_outputs/issue3333.p4

```diff
@@ -0,0 +1,16 @@
+enum bit<8> myenum {
```

Comment:
I think a problem with these tests is that they are all optimized away in the front end. So you can not actually see how these programs are transformed or how the mid end passes handle this. Even though most of these changes just check for errors.